src/goto-diff/goto_diff_parse_options.cpp

```diff
@@ -234,6 +236,12 @@ void goto_diff_parse_optionst::get_command_line_options(optionst &options)
             << " must not be given together" << eom;
     exit(1);
   }
+
+  if(cmdline.isset("show-properties"))
+    options.set_option("show-properties", true);
+  else
+    options.set_option("show-properties", false);
+
```

Comment:
`options.set_option("show-properties", cmdline.isset("show-properties"));` and no need for blank line before closing `}`